spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/cloud_manage_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager do
+  context ".raw_connect" do
+    let(:config) do
+      {
+        :provider               => "IbmCloud",
+        :IbmCloud_project         => "project",
+        :IbmCloud_json_key_string => "encrypted",
+        :app_name               => Vmdb::Appliance.PRODUCT_NAME,
+        :app_version            => Vmdb::Appliance.VERSION,
+        :Ibm_client_options  => { :proxy_url => "proxy_uri" },
```

Comment:
Minor (and miq-bot will also mention it) but we should align the `=>` and no spaces in the `{}`